src/components/DeclareLostDialog/DeclareLostInfo.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  Layout,
+  TextArea,
+  Row,
+  Col,
+} from '@folio/stripes/components';
+
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+import css from './DeclareLostDialog.css';
+
+class DeclareLostInfo extends React.Component {
+  static manifest = Object.freeze({
+    declareLost: {
+      type: 'okapi',
+      fetch: false,
+      POST: {
+        path: (_q, _p, _r, _l, props) => {
```

Comment:
I think you should be able to replace it with just `circulation/loans/!{loan.id}/declare-item-lost` instead of the function here. I wonder if you tried that pattern?

Reply:
Actually, I didn't know about this solution. It works great!

@mkuklis Thanks for you comments, the code looks better after changes!